concepticondata/conceptlists.tsv

```diff
@@ -295,3 +295,4 @@ Monnier-2014-1031	Monnier, Catherine and Syssau, Arielle	2014		1031	ranked	Frenc
 DiezAlamo-2018-750	Diez-Alamo, Antonio M. and Diez, Emiliano and Alonso, Maria Angeles and Vargas, C. Alejandra and Fernandez, Angel	2018		750	ranked	Spanish	Spanish	https://link.springer.com/article/10.3758/s13428-017-0970-y#SupplementaryMaterial	Diez2018		The study includes ratings of perceptual and motor attributes for a set of 750 concrete concepts in Spanish by requiring scale-based judgments from a sample of university students (N = 539). Values for body-object interaction scores were from [Alonso et al. (2018)](:bib:Alonso2018) Concreteness and imageability values were taken from the EsPal database [(Duchon et al., 2013)](:bib:Duchon2013). Information about the semantic category was included from [Marful et al., 2015](:bib:Marful2015).		
 Hsiu-2015-315	Hsiu, Andrew C.	2015		315	areal, questionnaire	English	Hmong-Mien languages	https://zenodo.org/record/1127804/	Hsiu2015		Hmong-Mien languages in Vietnam. The dataset contains 12 Varieties and 315 concepts.		
+Maciejewski-2016-100	Maciejewski, Greg and Klepousniotou, Ekaterini	2016		100	ranked	English	English	 http://osf.io/7k3eh/	Maciejewski2016		The list comprises relative meaning frequencies for 100 homonyms (British eDom Norms) and other lexical norms. For feasibility, each line of the table was duplicated to map the different meanings to one of the concepts found in Concepticon. The meaning definition was taken from the Wordsmyth dictionary [(Parks et al., 1998)](:bib:Parks1998). The dominance scores were included from [Armstrong et al. (2012)](:bib:Armstrong2012). Word frequencies and number of orthographic neighbours  were based on the British National Corpus [(BNC, 2007)](:bib:BNC2007). The numbers for word senses were taken from [Parks et al. (1998)](:bib:Parks1998) and WordNet [(Fellbaum, 1998)](:bib:Fellbaum1998). Semantic diversity values were included from [(Hoffman et al., 2013)](:bib:Hoffman2013). Ratings on familiarity, concreteness, and imageability were taken from [Stadthagen-Gonzalez and Davis (2006)](:bib:Stadthagen2006) and [Coltheart (1981)](:bib:Coltheart1981).		
```

Comment:
Small nitpicking: We tend to not use Author, Year, when in brackets, but simply Author Year, without a comma.